### PR TITLE
MAIN-5471 | Load dev config on devboxes at default

### PR DESCRIPTION
**Pull request by kvas-damian**
Load dev setting on devboxes at default
https://wikia-inc.atlassian.net/browse/MAIN-5471

Ping @rogatty @gabrys

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
     File should look something like this:
 ``` javascript
-    import devLocalSettings = require('./localSettings.dev');
     import Utils = require('../server/lib/Utils');
 
-    var localSettings = devLocalSettings.extendSettings({
+    var baseLocalSettingPath: string = process.env.WIKIA_ENVIRONMENT === 'dev' ? './localSettings.dev' : './localSettings.base',
+        baseLocalSettings = require(baseLocalSettingPath),
+        localSettings = baseLocalSettings.extendSettings({
         devboxDomain: 'joe',
```
Comment (rogatty) [inline README.md]: Indentation should be adjusted.

Comment (kvas-damian) [inline README.md]: Done!

File: config/localSettings.example.ts [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
  */
-import baseLocalSettings = require('./localSettings.base');
 import Utils = require('../server/lib/Utils');
 
-var localSettings = baseLocalSettings.extendSettings({
+var baseLocalSettingPath: string = process.env.WIKIA_ENVIRONMENT === 'dev' ? './localSettings.dev' : './localSettings.base',
+	baseLocalSettings = require(baseLocalSettingPath),
+	localSettings = baseLocalSettings.extendSettings({
 	// Add custom settings here
 });
```
Comment (rogatty) [inline config/localSettings.example.ts]: I think that formatting is broken here too :speak_no_evil:

File: README.md [hunk 3]
```diff
@@ -8,21 +8,21 @@
...
-    var localSettings = devLocalSettings.extendSettings({
-        devboxDomain: 'joe',
-        loggers: {
-            console: 'debug'
-        },
-        port: 8000 // 7000 if running on devbox
+    var baseLocalSettingPath: string = process.env.WIKIA_ENVIRONMENT === 'dev' ? './localSettings.dev' : './localSettings.base',
+        baseLocalSettings = require(baseLocalSettingPath),
+        localSettings = baseLocalSettings.extendSettings({
+            devboxDomain: 'joe',
```
Comment (gabrys) [inline README.md]: joe?

**Comment by bkoval** [comment]
LGTM :sandal:

**Pull request merged**